docs/units.rst

```diff
@@ -0,0 +1,19 @@
+=====
+Units
+=====
+
+Foyer uses the following units:
+
++----------+---------+
+| Quantity |  Units  |
++==========+=========+
+| distance |    nm   |
++----------+---------+
+|   angle  | radians |
++----------+---------+
+|   mass   |   amu   |
++----------+---------+
+|  energy  |  kJ/mol |
++----------+---------+
```

Comment:
Might be nice to link `OpenMM` [units](http://docs.openmm.org/latest/userguide/theory.html#units) here as well.

Reply:
good call!